transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -268,6 +276,8 @@ void bindInternal(final ResourceAddress address, IoHandler handler,
         //
         final ResourceAddress transportAddress = address.getTransport();
 
+        handshakeTimeoutByTransportAddress.put(transportAddress, address.getOption(HANDSHAKE_TIMEOUT).longValue());
```

Comment:
And where should the default value be set then? In case there is no user defined value in the config.